src/components/LogoBar.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { LogoBarStyle } from "./LogoBarStyle";
+
+export default function LogoBar() {
+  return (
+    <LogoBarStyle>
+      <embed src="RestaurantLogo.svg" />
```

Comment:
Don't use embed for this. You can create a folder for svgs and export them from there
<img width="942" alt="image" src="https://user-images.githubusercontent.com/65440011/180600460-defe9105-6db9-4d19-9e0d-2b3e6404e0fe.png">

Then you can either import them in styles file and style them with styled-components or you can use them as a component
<img width="568" alt="image" src="https://user-images.githubusercontent.com/65440011/180600500-6b074528-c664-4d69-85de-b9e1d68be78d.png">